### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by rismita87**
Databases week2 home work

### CODE REVIEW

File: databases/week2/main.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
+-- Part 1: Working with tasks
+-- 1.Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+SET
+  NAMES utf8mb4;
+USE hyfdbweek2;
+ALTER TABLE
```
Comment (mateusms01) [inline databases/week2/main.sql:6]: very well formatted long query. well done

File: databases/week2/main2.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---Part 2: School database
+-- Create a new database containing the following tables:
+CREATE DATABASE hyfdbweek2HW DEFAULT CHARACTER SET = 'utf8mb4';
+USE hyfdbweek2HW;
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE Class (
+  id int PRIMARY KEY,
```
Comment (mateusms01) [inline databases/week2/main2.sql:7]: good use of primary key

File: databases/week2/main.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "Homework",
+    "Databases week2 homework",
+    "2022-03-13 06:54:16",
+    "2022-03-15 06:54:16",
+    "2022-03-18 06:54:16",
+    "1",
+    "rismita87"
+  );
+--2.Change the title of a task
+UPDATE
```
Comment (mateusms01) [inline databases/week2/main.sql:31]: Formatting suggestion, I wouldn't split the UPDATE statement from the TABLE NAME it updates. 

Pointer: improves code readability and clarity

File: databases/week2/main3.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
+--Part 3: More queries
+SET
+  NAMES utf8mb4;
+USE hyfdbweek2;
+--1.Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT
+  task.title,
+  user.email
+FROM
+  task
```
Comment (mateusms01) [inline databases/week2/main3.sql:10]: formatting tip, would always keep from + table name on the same line. 
Pointer: improves readability and clarity

Comment (mateusms01) [reply databases/week2/main3.sql:10]: see other occurrences and  update

File: databases/week2/main4.sql [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---Part 4: Creating a database
```
Comment (mateusms01) [inline databases/week2/main4.sql:1]: well done here! really well elaborated-

File: databases/week2/main4.sql [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+CREATE DATABASE dbweek2HW4 DEFAULT CHARACTER SET = 'utf8mb4';
+USE dbweek2HW4;
+CREATE TABLE genre(
+  id INT PRIMARY KEY,
```
Comment (mateusms01) [inline databases/week2/main4.sql:5]: well done defining the PK!

**Review by mateusms01 — CHANGES_REQUESTED** [review]
Minor comments added, really well done!

**Comment by rismita87** [comment]
Hi Mateus,
thanks a lot for your feedback. It would be really helpful if you can give
some feedback on the entity relationship diagram (pdf file) as well.
Thanks & Regards

**Comment by mateusms01** [comment]
Hey Rismita, really sorry for that I completely overlooked the diagram. 

In general the diagram is great, and well defined. However, there is two tables that are not necessary.

Both Genre-Book and Books-Racks are Many-to-One and not Many-to-Many, therefore they could be directly linked without intermediate tables.

**Comment by rismita87** [comment]
Hi Mateus, thanks for your feedback.
Regards
Rismita.

**Pull request merged**